src/internet_identity/tests/api/mod.rs

```diff
@@ -1,9 +1,8 @@
 /** The functions here are derived (manually) from Internet Identity's Candid file */
-use crate::framework;
-use crate::framework::CallError;
-use candid;
 use candid::Principal;
-use ic_state_machine_tests::{CanisterId, PrincipalId, StateMachine};
+use canister_tests::framework::*;
```

Comment:
let's avoid global imports, they really hurt readability

Reply:
meh, actually I don't care

Reply:
Well, I have now reverted it anyways. In my opinion, having wildcard imports for _local_ dependencies is fine.

Maybe we can revisit this topic some other time.